com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -115,21 +118,21 @@ static void DrawConfinerGizmos(CinemachineConfiner2D confiner2D, GizmoType type)
                 return;
 
             var color = CinemachineCorePrefs.BoundaryObjectGizmoColour.Value;
-            var colorDimmed = new Color(color.r, color.g, color.b, color.a / 2f);
+            var colorDimmed = new Color(color.r, color.g, color.b, color.a / 0.5f);
```

Comment:
dividing by 0.5 is like multiplying b 2.  You are making it brighter, unless it's already 1 in which case you're doing nothing

Reply:
That's a typo :) not even sure how it got changed.

The color is up for discussion. Julien will get back to me.

Reply:
I think the inner polygon should be dimmer

Reply:
Yes, that's how it was originally.